.wpilib/wpilib_preferences.json

```diff
@@ -2,5 +2,5 @@
     "enableCppIntellisense": false,
     "currentLanguage": "java",
     "projectYear": "2022",
-    "teamNumber": 2473
+    "teamNumber": 2475
```

Comment:
?

Reply:
We changed the old robot to have team number 2475 (so the ip for the router is different). So to test code on that robot, we needed to change the team number to 2475 (prob shouldn't have pushed that change to Drive though).